src/services/user/model/consent.schema.js

```diff
@@ -27,10 +27,7 @@ const consentSchema = new Schema({
 		privacyConsent: { type: Boolean },
 		termsOfUseConsent: { type: Boolean },
 	}],
-});
-
-enableAuditLog(consentSchema);
```

Comment:
Why does the hotfix change audit log behavior for the consent schema?

Reply:
This is a side effect of not being a schema anymore. It is part of the user schema.